v1.1/js/Node.js

```diff
@@ -252,7 +252,8 @@ function Node(model, config){
 			width = ctx.measureText(self.label).width;
 		}
 		ctx.fillText(self.label, 0, 0);
-
+		var roundedValue = Math.round(self.value * 100) / 100; // Temp variable solely to cleanly display value
```

Comment:
It boggles my mind that apparently there aren't any overloads for Math.round() to specify the number of decimal places to round to in Javascript